full-node/db/sov-schema-db/src/lib.rs

```diff
@@ -292,58 +62,101 @@ impl SchemaBatch {
         key: &impl KeyCodec<S>,
         value: &impl ValueCodec<S>,
     ) -> anyhow::Result<()> {
-        let _timer = SCHEMADB_BATCH_PUT_LATENCY_SECONDS
+        #[cfg(feature = "std")]
+        let _timer = metrics::SCHEMADB_BATCH_PUT_LATENCY_SECONDS
             .with_label_values(&["unknown"])
             .start_timer();
+
         let key = key.encode_key()?;
         let value = value.encode_value()?;
+
+        #[cfg(feature = "std")]
         self.rows
             .lock()
             .expect("Lock must not be poisoned")
             .entry(S::COLUMN_FAMILY_NAME)
             .or_default()
             .push(WriteOp::Value { key, value });
 
+        #[cfg(not(feature = "std"))]
```

Comment:
we have `cfg_attr(not(feature = "std"), allow(dead_code))]` if it is not used in `std` why do we have the flag here?